pkg/health/health.go

```diff
@@ -35,18 +35,29 @@ var felixLivenessEp string
 
 func init() {
 	felixPort := os.Getenv("FELIX_HEALTHPORT")
-	if felixPort == "" {
+	felixHost := os.Getenv("FELIX_HEALTHHOST")
+	if felixPort == "" && felixHost == "" {
 		felixReadinessEp = "http://localhost:9099/readiness"
 		felixLivenessEp = "http://localhost:9099/liveness"
 		return
+
+	} else if felixPort == "" {
+		felixReadinessEp =  "http://" + felixHost + ":9099/readiness"
+		felixLivenessEp = "http://" + felixHost + ":9099/liveness"
+		return
+
+	} else if felixHost == "" {
+		felixReadinessEp = "http://localhost:" + felixPort + "/readiness"
+		felixLivenessEp = "http://localhost:" + felixPort + "/liveness"
+		return
 	}
```

Comment:
Thank you for submitting this.
What do you think about changing this a bit?
I think instead of dealing with the 4 different cases (L40, L45, L50, and L59) it would be good to do something like
```
felixPort := os.Getenv("FELIX_HEATHPORT")
if felixPort == "" {
  felixPort = "9099"
}
if _, err := strconv.Atoi(felixPort); err != nil {
￼  log.Panicf("Failed to parse value for port %q", felixPort)
￼}

felixHost := os.Getenv("FELIX_HEALTHHOST")
if felixHost == "" {
  felixHost = localhost
}
felixReadinessEp = "http://" + felixHost + ":" + felixPort + "/readiness"
felixLivenessEp = "http://" + felixHost + ":" + felixPort + "/liveness"
```

Reply:
Yeah, your proposition makes more sense. Will change that in a moment.